docs/theme/components/Menu.styles.ts

```diff
@@ -1,10 +1,33 @@
 import { ThemeComponent, cva } from '@marigold/system';
 
 export const Menu: ThemeComponent<'Menu'> = {
-  container: cva([
-    'border-border bg-bg-surface overflow-hidden rounded-md border px-1 py-1.5 text-sm shadow-md',
-    'sm:max-h-[45vh] md:max-h-[75vh] lg:max-h-[75vh]',
-  ]),
-  item: cva('focus:bg-bg-hover cursor-pointer p-2 outline-none'),
-  section: cva(''),
+  container: cva(
+    [
+      'border-border bg-bg-surface overflow-hidden rounded-md border px-1 py-1.5 text-sm shadow-md',
+      'sm:max-h-[45vh] md:max-h-[75vh] lg:max-h-[75vh]',
+    ],
+    {
+      variants: {
+        variant: {
+          command:
+            '[&_[cmdk-list-sizer]]:divide-secondary-100 size-full px-4 sm:w-[500px] [&_[cmdk-input-wrapper]_svg]:size-5 [&_[cmdk-input]]:h-12 [&_[cmdk-item]]:py-2 [&_[cmdk-item]_svg]:size-5 [&_[cmdk-list-sizer]]:divide-y',
+        },
+      },
+    }
+  ),
+  item: cva('focus:bg-bg-hover cursor-pointer rounded p-2 outline-none', {
+    variants: {
+      variant: {
+        command: ['aria-selected:bg-bg-hover py-1.5'],
```

Comment:
Some styles are missing now or? For example when I hover over an item the font-weight doesn't change. Or is it intended?

Reply:
it was intented to align the Menu and Command styles.